core/templates/domain/state/StateObjectFactorySpec.ts

```diff
@@ -174,7 +178,9 @@ describe('State Object Factory', () => {
   it('should create a default state object', () => {
     const stateName = 'Default state';
     const stateObjectDefault = sof.createDefaultState(stateName);
-    const outcome = <OutcomeBackendDict>stateObject.interaction.default_outcome;
+
+    const outcome = (
+      stateObject.interaction.default_outcome as DefaultOutcome);
```

Comment:
Created a new type here because `default_outcome` is of type `OutcomeBackendDict | null` causing next line to throw error.`Object is possibly null`.